### PR TITLE
Deletion require confirmation - special method execution is required

### PR DESCRIPTION
**Pull request by mrzygood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,20 @@ yarn add flowchart-vue
...
-                    @save="handleChartSave" ref="chart">
+        <button type="button" v-if="showRemovingConfirmation" @click="confirmRemoving">
+            Confirm removing
+        </button>
+        <button type="button" v-if="showRemovingConfirmation" @click="showRemovingConfirmation = false">
+            Reject removing
+        </button>
+        <flowchart :nodes="nodes" 
+                   :connections="connections" 
+                   :removeRequiresConfirmation
```
Comment (iamppz) [inline README.md]: Use kebab-case to align with HTML attributes: https://vuejs.org/guide/components/props.html#prop-passing-details

File: README.md [hunk 2]
```diff
@@ -29,9 +29,20 @@ yarn add flowchart-vue
...
+                   @editnode="handleEditNode"
+                   @dblclick="handleDblClick" 
+                   @editconnection="handleEditConnection"
+                   @removeConfirmationRequired="initRemovingConfirmation"
```
Comment (iamppz) [inline README.md]: Use lowercase to align with other props

File: README.md (hunk 1 above)
Comment (iamppz) [reply README.md]: Value of `removeRequiresConfirmation ` is missing

File: README.md [hunk 3]
```diff
@@ -29,9 +29,20 @@ yarn add flowchart-vue
         <button type="button" @click="$refs.chart.save()">
             Save
         </button>
-        <flowchart :nodes="nodes" :connections="connections" @editnode="handleEditNode"
-                    @dblclick="handleDblClick" @editconnection="handleEditConnection" 
-                    @save="handleChartSave" ref="chart">
+        <button type="button" v-if="showRemovingConfirmation" @click="confirmRemoving">
```
Comment (iamppz) [inline README.md:32]: Can't understand the meaning of `@click="confirmRemoving"`, maybe it should be `initRemovingConfirmation` or `showRemovingConfirmation = true`?

File: README.md (hunk 1 above)
Comment (mrzygood) [reply README.md]: Ok

File: README.md (hunk 3 above)
Comment (mrzygood) [reply README.md:32]: Actually it is confirmation - you decides to agree that you are sure you want to delete elements.
Wrong value was in `showRemovingConfirmation`.

File: README.md (hunk 1 above)
Comment (mrzygood) [reply README.md]: Solved.

File: README.md (hunk 2 above)
Comment (mrzygood) [reply README.md]: solved.

File: README.md (hunk 3 above)
Comment (mrzygood) [reply README.md:32]: Solved.

**Comment by mrzygood** [comment]
Updates regarding your comments have appeared.

**Comment by iamppz** [comment]
> Updates regarding your comments have appeared.

Thanks, I’ll publish it later :)

**Pull request merged**